style.css

```diff
@@ -650,6 +620,136 @@ article h1 {
     outline: none;
     height: 70px;
   }
+
+  .modal {
+    background-color: #e5e5e5;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    overflow: scroll;
+  }
+
+  .hidden-modal {
+    display: none;
+  }
+
+  .modal-box {
+    margin: 5%;
+    width: 70%;
+    height: 120%;
+    background-color: #fff;
+  }
+
+  .modal img {
+    width: 80%;
+    height: 500px;
+    margin-left: 10%;
+    z-index: -7;
+  }
+
+  .modal-box div {
+    height: 50px;
+    width: 100%;
+    padding-right: 40px;
+  }
+
+  .modal .close {
+    font-size: 50px;
+    width: 50px;
+    height: 50px;
+    color: #172b4d;
+    background-color: #e5e5e5;
```

Comment:
- [ ] Kindly correct the background color from the close button as the last code reviewer suggested 👇🏽 

> I would recommend you stick to the style guide of the Figma design esp. the mobile responsiveness. At the moment, your popup is not following the font, margin, and padding styles.